dockers/docker-fpm-frr/frr/bgpd/templates/general/policies.conf.j2

```diff
@@ -50,6 +50,7 @@ route-map FROM_BGP_PEER_V6 permit 11
 route-map FROM_BGP_PEER_V4 permit 100
 !
 route-map TO_BGP_PEER_V4 permit 100
+ call CHECK_ORIGINATE_BANDWIDTH_ENABLED
```

Comment:
@colintle is it safe to call `route-map` when it is not defined yet? Imagine `init_cfg/config_db` is missing the entry

Reply:
We are doing the same with idf_isolation: https://github.com/sonic-net/sonic-buildimage/pull/19136/files#diff-65eae5be47ac26f0b10080ed57e8395630c2ef0f44aaf3b256d52f37677f38ae